setup.cfg

```diff
@@ -39,7 +39,7 @@ install_requires =
     pytz>=2021.3
     requests>=2.26.0
     ccxt>=1.81.30
-    rp2>=1.0.8
+    rp2>=1.0.9
```

Comment:
Leave an empty line after last dependency. Also can you move ccxt up so that it's in alphabetical order?